stockpile/roles/openshift-cluster-topology/tasks/main.yml

```diff
@@ -42,6 +42,31 @@
       shell: oc whoami
       register: ocp_user
 
+    - name: copy the openshift_config_scraper script
+      copy:
+        src: openshift_config_scraper.py
```

Comment:
Use script module perhaps? and also add a when condition such that it tries to collect only if openshift is installed

Reply:
The tasks are grouped under a block and the block is run only when the oc client is installed and if a kubeconfig exists.

Reply:
Ack, just saw it.

Reply:
Absolutely.